src/beignet/_bisect.py

```diff
@@ -0,0 +1,59 @@
+from dataclasses import dataclass
+from typing import Callable, Literal
+
+import torch
+from torch import Tensor
+
+from beignet._root_scalar import RootSolutionInfo
```

Comment:
Use relative imports, i.e., `from ._root_scalar import RootSolutionInfo`.

Reply:
What is the reason to prefer one over the other? I am never quite sure which to use